bonsai-pay/deployment-guide.md

```diff
@@ -0,0 +1,172 @@
+# RISC Zero Ethereum Deployment Guide
+
+> **Note: This software is not production ready. Do not use in production.**
+
+Welcome to the [RISC Zero] Ethereum Deployment guide!
+
+Once you've written your [contracts] and your [methods], and [tested] your program, you're ready to deploy your contract.
+
+You can either:
+
+- [Deploy your project to a local network]
+- [Deploy to a testnet]
+
+## Deploy your project on a local network
+
+You can deploy your contracts and run an end-to-end test or demo as follows:
+
+1. Start a local testnet with `anvil` by running:
+
+    ```bash
+    anvil
+    ```
+
+    Once anvil is started, keep it running in the terminal, and switch to a new terminal.
+
+2. Set your environment variables:
+    > ***Note:*** *This requires having access to a Bonsai API Key. To request an API key [complete the form here](https://bonsai.xyz/apply).*
+
+    ```bash
+    # Anvil sets up a number of default wallets, and this private key is one of them.
+    export ETH_WALLET_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+    export BONSAI_API_KEY="YOUR_API_KEY" # see form linked in the previous section
+    export BONSAI_API_URL="BONSAI_API_URL" # provided with your api key
+    ```
+
+3. Build your project:
+
+    ```bash
+    cargo build
+    ```
+
+4. Deploy your contract by running:
+
+    ```bash
+    forge script --rpc-url http://localhost:8545 --broadcast script/Deploy.s.sol
```

Comment:
I run into an issue with this because the git submodules aren't registered. Tried running `git submodule update --init` and it didn't pull the ones defined in this dir. Did you run this outside of this repo and copy it over? Perhaps the libs dir should be off gitignore?

Reply:
oh, so how did you have this install, for devs who want to build on this?

Reply:
Add the submodules - `git submodule update --init` should work now. 
 monorepos :(